tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/NativeDesktop/NativeDesktopThirdParty/Supplier Get Native Desktop Third Party.feature

```diff
@@ -32,7 +32,7 @@ Scenario: Native Desktop Third Party is retrieved for the solution where no nati
     And the device-capabilities string does not exist
 
 @3621
-Scenario: Native Desktop Third Party is retrieved for the solution where no solution detail exists
+Scenario: Native Desktop Third Party is retrieved empty for the solution
     When a GET request is made for native-desktop-third-party section for solution Sln3
     Then a successful response is returned
     And the third-party-components string does not exist
```

Comment:
I think we can remove this scenario as it looks like it is already covered by the scenario above – it might be worth double-checking if any of the other changed scenarios are now redundant too.

Reply:
Removed duplicate scenario.